cubic-server/command_parser/commands/Op.cpp

```diff
@@ -0,0 +1,34 @@
+#include "Op.hpp"
+#include "Server.hpp"
+
+using namespace command_parser;
+
+void Op::autocomplete(std::vector<std::string>& args, Player *invoker) const {
+}
+
+void Op::execute(std::vector<std::string>& args, Player *invoker) const {
+    if (invoker && !invoker->isOperator())
+        return;
+    if (args.empty()) {
+        LINFO("Need a player name");
+    } else {
+        if (args.size() != 1)
```

Comment:
Same

Reply:
Is the goal to reproduce the exact same behavior as the op command ? If not, you could support multiple op at once `/op [username1] [usernameX...]`

Reply:
I reproduced the same behavior of the op command of the official server so it cannot give operator privileges to multiple players.